public/index.html

```diff
@@ -14,6 +14,7 @@
   <title>Windows 11 in React</title>
   <link rel="stylesheet" href="dycalendar.css">
   <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"/>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
```

Comment:
was this pr even reviewed?

Reply:
LMAO, I didn't notice that...

Reply:
The CSS sheet is pointless, isn't it? Probably is slowing it down...